### DISCUSSION TITLE
Bump Fee CPFP Tx also stuck, try again?

### DISCUSSION THREAD

**Discussion by imyjimmy** [Ride-The-Lightning/RTL#1328] [Q&A] [answered]
I recently attempted to bump a channel open txn. I didn't know what CPFP is prior to this and thought the new fee I entered would be the fee for the txn. 

Turns out CPFP takes something like the weighted average of the two. Knowing this, I now want to bump the fee again. I’m trying to figure out what would happen if I do a CPFP fee bump twice. Assuming I set a high enough fee this time, my guess as to what would happen: the second CPFP tx would drag the original tx through along w it, leaving the first CPFP tx stuck in the mempool. At a later point in time the first CPFP transaction would get mined, which is effectively a payment to myself. Am i getting this?

**Comment by saubyk** [comment, marked as answer]
> At a later point in time the first CPFP transaction would get mined, which is effectively a payment to myself

Hi @imyjimmy this understanding is incorrect

If you cpfp a second time and the transaction gets mined, the parent (1st cpfp) and the grandparent transaction (pending channel open) would get mined along with it. Of course for this to happen, the fee rate on the second cpfp has to be set high enough such that the **cumulative fee rate** for all the three transactions is competitive
